### ISSUE TITLE
Reduced CD95-induced lymphocyte apoptosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6465]
**Preferred term label:**
Reduced CD95-induced lymphocyte apoptosis

**Synonyms**
Defective CD95-induced lymphocyte apoptosis

**Definition (free text, please give PubMed ID)**
Reduced amount of programmed cell death in peripheral blood lymphocytes following exposure to CD95 (APO-1/Fas), which is a member of the death receptor family, a subfamily of the TNF-R superfamily. Crosslinking of CD95 with its natural ligand CD95L (CD178) or with agonistic antibodies such as anti-APO-1 induces apoptosis in sensitive cells. PMID:21625644

**Parent term (use hpo.jax.org/app)**
Decreased lymphocyte apoptosis
https://hpo.jax.org/app/browse/term/HP:0002731

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
607271 
https://pubmed.ncbi.nlm.nih.gov/12353035/  (2/2)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
